### ISSUE TITLE
Bright flash of light upon opening a new tab

### ISSUE THREAD

**Issue by haIfway** [brave/brave-browser#16078]
Each time a new tab is opened on brave, a brief flash of bright light appears. This could be dangerous for people who have optic nerve problems or seizures. 

Here is a demo of the issue: https://www.youtube.com/watch?v=5F7p2j311vk
This is similar to a previous closed issue: https://github.com/brave/brave-browser/issues/3575

## Brave version (brave://version info)

Brave | 1.24.86 Chromium: 90.0.4430.212&nbsp;(Official Build)&nbsp;(64-bit)
-- | --
Revision | e3cd97fc771b893b7fd1879196d1215b622c2bed-refs/branch-heads/4430@{#1429}
OS | Linux
JavaScript | V8&nbsp;9.0.257.29
User Agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36
Command Line | /opt/brave.com/brave/brave --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --enable-features=AutoupgradeMixedContent,SafetyTip,DnsOverHttps,LegacyTLSEnforced,WebUIDarkMode,PasswordImport,ReducedReferrerGranularity,PrefetchPrivacyChanges --disable-features=TextFragmentAnchor,EnableProfilePickerOnStartup,AutofillServerCommunication,NotificationTriggers,WebOTP,SignedExchangePrefetchCacheForNavigations,SubresourceWebBundles,AutofillEnableAccountWalletStorage,TabHoverCards,DirectSockets,SharingQRCodeGenerator,LangClientHintHeader,NetworkTimeServiceQuerying,IdleDetection,SafeBrowsingEnhancedProtectionMessageInInterstitials,SignedExchangeSubresourcePrefetch,SafeBrowsingEnhancedProtection --flag-switches-begin --flag-switches-end
Executable Path | /opt/brave.com/brave/brave
Profile Path | /home/a/.config/BraveSoftware/Brave-Browser/Default
Variations | aed3cac0-a9b0841d7146a73c-3f4a17df

OS: Ubuntu 20.04.2 LTS

## More Information
Related issue on Android: https://github.com/brave/brave-browser/issues/11185

**Comment by rebron** [comment]
@haIfway Do you have extension running that replaced your new tab page? If you disable that temporarily, does that fix the bright flash?

**Comment by haIfway** [comment]
It reduces the bright flash, however it's still there.

demo: https://www.youtube.com/watch?v=wwYFTQe7onQ

**Comment by Kahana82** [comment]
This is old (chromium 57) but may still be of interest to the devs in regard to this problem:
[https://github.com/hbtlabs/chromium-white-flash-fix](https://github.com/hbtlabs/chromium-white-flash-fix)

If I understood the above correctly, it looks like there is a way to change the very first color a page/tab/window is filled with before any content is rendered into it.

It would be nice to have that color be **non-white** when using the `Dark` option for `Brave colors` in the `Appearance` section of the browser's settings.
The dark background color used by the Dark theme would be ok, but some people might still prefer black or even a custom value.

**Comment by rmdhnreza** [comment]
I think only Vivaldi has resolved this issue. Brave, Chrome, Edge (Browser I installed) still have this issue.

1. Set Brave colors to `Dark`
2. Enable `brave://flags/#enable-force-dark` Flags
3. Browsing
4. White Flash appears

[Video](https://i.imgur.com/pKwMSTK.mp4)

**Comment by thehotery** [comment]
Same and very annoying issue with Brave on Windows with Speed Dial [FVD] extension enabled as new tab page

**Comment by Brave-Matt** [comment]
+1 from Community:
https://community.brave.com/t/brave-dark-mode-blank-screen/266713/2

**Comment by Brave-Matt** [comment]
+1 from Community:
https://community.brave.com/t/white-flash-when-opening-hyperlinks-or-new-tabs-on-websites-that-have-dark-reader-disabled/266112/5

**Comment by sdgrs546547Q** [comment]
one more example: https://imgur.com/a/LRyL9df

**Comment by bugbountyguy** [comment]
Plus 1 to this issue. I've recently had to change my system to a global 'dark theme' setup due to some personal issues with my eyes. This is a big thing for a11y and honestly should be a high priority, in my opinion... I bet Brave could attract the a11y community who fall under the same category as myself if they fixed this white flash issue between tabs and page transitions (since not even Google Chrome has fixed this yet and it's been known for _years_).

**Comment by sdgrs546547Q** [comment]
please fix this. In browsers like Firefox this doesnt happen

**Comment by racofer** [comment]
Still waiting on this one so that I can get back into using Brave. It's fixable in Firefox with about:config, and Vivaldi already fixed it some time ago.

Having flashbangs thrown at me while browsing is not fun.

**Comment by Tiharo** [comment]
I might be assuming this, but I believe this would be a quick fix to make a lot of people's browsing experience a little bit better.

**Comment by zoombinis** [comment]
@rebron I noticed this issue is labeled: `needs-more-info`. I'd like to help, what info is still needed that the community can provide?

**Comment by thehotery** [comment]
Some time ago (and couple of chromium versions before) its almost dissapear on all chromium-based browsers, or at least became so fast that eye can barely catch it. I'm also user of "FVD Speed Dual" extension and dark-reader extension, and Dark Reader extension and when they both are disabled, flashes became much shorter, but not dissapear completely... Maybe its something depending of gpu, because on another pc with intel graphics flashes are shorter then on nvidia.. 

_hope this issue will not be closed because this problem so important for many peoples that like dark mode, although the problem, of course, affects common for all chromium-browsers, not brave itself_

**Comment by spylogsster** [comment]
In order to fix this we have to add API in the new tab page to rotate new tab background images smoothly without white flash. Created issue for this https://github.com/brave/brave-browser/issues/20508, when that will be fixed we can update the new tab to rotate backgrounds and open new ones beside the current one

**Comment by koodinmies** [comment]
Is there a way to change the     
`--default-bg-color: white;`
to
`--default-bg-color: black;`

in
chrome://newtab or brave://newtab

I'm not sure but I think that should fix the issue.

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.42.62 Chromium: 103.0.5060.114 (Official Build) beta (x86_64)
-- | --
Revision | a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}
OS | macOS Version 13.0 (Build 22A5295i)

Confirmed the issue using `1.41.96`; confirmed the fix changing the background color to a dark grey using `1.42.62`

`1.41.96` | `1.42.62`
----------|----------
![release-build-flash](https://user-images.githubusercontent.com/387249/178857974-90c7ec4f-64c0-474f-bf97-5e5f9dc93d6e.gif) | ![beta-build-no-flash](https://user-images.githubusercontent.com/387249/178857995-8f476d4d-78c0-41ed-93af-7107313ee6c9.gif)

**Comment by dswapnil104** [comment]
> I think only Vivaldi has resolved this issue. Brave, Chrome, Edge (Browser I installed) still have this issue.
> 
> 1. Set Brave colors to `Dark`
> 2. Enable `brave://flags/#enable-force-dark` Flags
> 3. Browsing
> 4. White Flash appears
> 
> [Video](https://i.imgur.com/pKwMSTK.mp4)

I continued to blame my hardware until I decided to look it up.
Thank you for the quick and dirty fix, a real eyesaver for me and other night owls.

**Comment by illtellyoulater** [comment]
So... what the solution was? I am still observing this problem with latest Brave (1.47.171 stable)...

@stephendonner how did you change the background color to a dark grey?

**Comment by stephendonner** [comment]
> So... what the solution was? I am still observing this problem with latest Brave (1.47.171 stable)...
> 
> @stephendonner how did you change the background color to a dark grey?

@illtellyoulater Nothing special; just followed the steps you see listed in the verification notes above.

Can you please file a new issue with a screencast, and cross-reference that new issue #, here?  Thanks!

**Comment by v3ss0n** [comment]
Solution so far for every chrome browser for me is to enable dark mode by default , in chrome:flags .

**Comment by stephendonner** [comment]
> @stephendonner yes but how did you change the background color to a dark grey?

@illtellyoulater as I mentioned, I didn't do anything different from the steps, and didn't experience this issue when verifying it.

You still experiencing this issue, now, means we need a new issue logged so we can investigate.

**Comment by illtellyoulater** [comment]
@stephendonner I understand but looking at the steps I can't see how to switch background to dark grey...

**Comment by AlexBilozir** [comment]
@illtellyoulater use "enabled" option
![image](https://user-images.githubusercontent.com/8397468/219068744-bc158197-01fc-4399-8e33-15270ecfc780.png)

**Comment by illtellyoulater** [comment]
oh stephendonner said dark grey specifically so I thought it was a specific setting among other possible dark tonalities.

**Comment by einarpersson** [comment]
I am still seeing this, and I am unable to disable it. 

I have Dark mode enabled. Enabling the Auto Dark Mode for Web Contents did not make a difference to this issue, however it had a lot of unwanted side-effects. I just want one (1) thing, which is to style/darken the white flash screen. Nothing else.

```
Version 1.59.124 Chromium: 118.0.5993.117 (Official Build) (64-bit)
Windows 10 Pro
```

**Comment by AwakeningLightning** [comment]
Still flash on startup on Windows 11
Found some info which might be useful:
https://www.reddit.com/r/chrome/comments/squ71r/comment/i932tml
